engine/jsonlint/pipe.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://repometric.github.io/linterhub/schema/extr.json",
+    "id": "jsonlint",
+    "name": "jsonlint",
+    "pipeline": [
+        {
+            "cmd": "jsonlint",
+            "success": 1,
+            "engine": true,
+            "args":{
+                "":"--compact",
+				"2>&1":""
```

Comment:
Is it an engine argument?

Reply:
No, but this argument to need correct work of linter.

Reply:
Hmm, then each step should have 2 args (`stdin` and smth else). let me think about it.

Reply:
This question is still not solved.